[docs] Move file-based metastore doc to installation/deployment.rst (#24620)
prestodb/presto: presto-docs/src/main/sphinx/installation/deployment.rst

    ./bin/launcher start

> File-Based MetaStore

Reviewer: nit: 
```suggestion
File-Based Metastore
```

--------------------

> For testing or developing purposes, Presto can be configured to use a local 

Reviewer: nit: would development be better than developing?

filesystem directory as a Hive Metastore. 

The file-based metastore works only with the following connectors: 

* :doc:`/connector/deltalake`
* :doc:`/connector/hive`
* :doc:`/connector/hudi`
* :doc:`/connector/iceberg`

Configuring a File-Based Metastore
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

1. In ``etc/catalog/``, find the catalog properties file for the supported 
   connector. 

2. In the catalog properties file, set the following properties:

.. code-block:: none

    hive.metastore=file
    hive.metastore.catalog.dir=file:///<catalog-dir>

Replace ``<catalog-dir>`` in the example with the path to a directory on an 
accessible filesystem.

Using a File-Based Warehouse
^^^^^^^^^^^^^^^^^^^^^^^^^^^^

For this example, assume the Hive connector is being used, and the properties 
in the Hive connector catalog file are set to the following:

.. code-block:: none

    connector.name=hive
    hive.metastore=file
    hive.metastore.catalog.dir=file:///data/hive_data/

Create a schema

.. code-block:: none

    CREATE SCHEMA hive.warehouse;

> This query creates a directory ``warehouse`` in the directory set for 
> ``hive.metastore.catalog.dir``, so the path to the new directory is 
> ``/data/hive_data/warehouse``.

Reviewer 2: I think we can rephrase this to something like -

```suggestion
This query creates a directory ``warehouse`` in the directory set for 
``hive.metastore.catalog.dir``, so the path for `warehouse` schema would be 
``/data/hive_data/warehouse``.
```
